lib/perl/Genome/InstrumentData/Command/Import/Basic_bam_multi_groups.t

```diff
@@ -86,7 +90,7 @@ for my $rg_id ( sort keys %expected_read_groups ) {
     ok($allocation, 'got allocation');
     ok($allocation->kilobytes_requested > 0, 'allocation kb was set');
 
-    #print join("\t", $rg_id, $paired_key, $instrument_data->bam_path), "\n"; <STDIN>;
 }
 
+#print join("\n", map { $_->bam_path } values %instrument_data)."\n"; <STDIN>;
```

Comment:
One option for `print`s like these might be to make them `debug_message`s.  Then they can be toggled with `UR_COMMAND_DUMP_DEBUG_MESSAGES`.  `note` or `diag` would also work in the context of the test.